app.py

```diff
@@ -81,6 +83,7 @@ def writeBanlist():
 def addToBanlist(fhash, fname, reason):
     banlist.append({'hash':fhash, 'filename':fname, 'reason':reason})
```

Comment:
Wouldn't it be better to put reloadBanlist as the first line here? Since if the file has been modified externally, the changes will be discarded and will be overwritten by the banlist we have in memory.